src/transformers/modeling_reformer.py

```diff
@@ -1817,3 +1817,99 @@ def prepare_inputs_for_generation(self, input_ids, past, **kwargs):
             inputs_dict["num_hashes"] = kwargs["num_hashes"]
 
         return inputs_dict
+
+
+class ReformerClassificationHead(nn.Module):
+    """Head for sentence-level classification tasks."""
+
+    def __init__(self, config):
+        super().__init__()
+        self.dense = nn.Linear(config.hidden_size, config.hidden_size)
```

Comment:
Here we have to be careful in Reformer because it uses reversible revnets. This means that self.dence should be `nn.Linear(2 * config.hidden_size, config.hidden_size)`

Reply:
Noted, will make the necessary change